html/elements/input/range.json

```diff
@@ -0,0 +1,62 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-range": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/range",
+            "support": {
+              "chrome": {
+                "version_added": "4"
+              },
+              "chrome_android": {
+                "version_added": "57",
+                "notes": "Chrome implements the <code>slider-vertical</code> value for the non-standard <code>-webkit-appearance</code> property. You shouldn't use this, since it's proprietary, unless you include appropriate fallbacks for users of other browsers."
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "23",
+                "notes": [
+                  "While the specification says that the range input should be drawn vertically if the height is greater than the width, this behavior is not currently implemented. See these Firefox bugs for more information: <a href='https://bugzil.la/840820'>bug 840820</a> and <a href='https://bugzil.la/981916'>bug 981916</a>.",
+                  "Drawing of hash/tick marks is not yet implemented. See <a href='https://bugzil.la/841942'>bug 841942</a> for more information."
+                ]
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": "11"
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": "3.1"
+              },
+              "safari_ios": {
+                "version_added": "5.1"
+              },
+              "webview_android": {
+                "version_added": "2.1",
+                "notes": "The Android browser recognizes the <code>range</code> type starting with version 2.1, but doesn't fully implement it until version 4.3."
```

Comment:
This may seem a little silly, but I'd replace `The Android browser` with "WebView" or "Android WebView" as to not imply that there's only one browser available for Android

Reply:
👍